pyerrors/correlators.py

```diff
@@ -343,6 +344,36 @@ def GEVP(self, t0, ts=None, sort="Eigenvalue", **kwargs):
         else:
             return reordered_vecs
 
+    def error_gevp(self, t0):
```

Comment:
While I usually prefer lower case method names, we already have a method called `GEVP`. Should we capitalize here as well? Not a strong opinion on this.

Reply:
I do not even have a weak opinion. You decide.